<launch>
  <arg name="gui" default="true" />

  <include file="$(find mashes_measures)/launch/measures.launch">
    <arg name="gui" value="$(arg gui)" />
  </include>

  <node name="control" pkg="mashes_control" type="nd_control.py">
    <param name="power_min" type="double" value="0.0" />
    <param name="power_max" type="double" value="1500.0" />
  </node>

  <node name="plot_power" pkg="rqt_plot" type="rqt_plot" args="/control/power/value" if="$(arg gui)" />

  <node name="labjack" pkg="mashes_labjack" type="nd_labjack.py">
    <param name="power_min" type="double" value="0.0" />
    <param name="power_max" type="double" value="1500.0" />
  </node>

  <node name="control_panel" pkg="mashes_control" type="qt_control.py" if="$(arg gui)" />
</launch>
